fundamentals/fcsreqs.tex

```diff
@@ -26,6 +26,12 @@ \subsubsection{Discrete Facilities and Materials}
 
 \subsubsection{Open Access}
 
+% gonuke: I am perhaps oversensitive, but I perceive the status of 
+%         VISION to be constantly shifting.  I am thus unsure whether
+%         ``license agreemnts or instutional approval'' are currently
+%         required for access to VISION.  I raise it here to satisfy
+%         my desire to have it categorically confirmed by someone.
```

Comment:
The powersim license was ~$800 if I recall correctly

On Sun, Feb 22, 2015 at 1:31 PM, Katy Huff <notifications@github.com> wrote:

> In fundamentals/fcsreqs.tex
> <https://github.com/cyclus/fundamentals-paper/pull/82#discussion_r25135501>
> :
>
> > @@ -26,6 +26,12 @@ \subsubsection{Discrete Facilities and Materials}
> >
> >  \subsubsection{Open Access}
> >
> > +% gonuke: I am perhaps oversensitive, but I perceive the status of
> > +%         VISION to be constantly shifting.  I am thus unsure whether
> > +%         ``license agreemnts or instutional approval'' are currently
> > +%         required for access to VISION.  I raise it here to satisfy
> > +%         my desire to have it categorically confirmed by someone.
>
> This is a good point. I think at this point, you do still have to ask them
> for it, but you may not have to do too much paperwork. Ultimately though,
> it relies on powersim, which I think does require a significant license,
> doesn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/pull/82/files#r25135501>.
>



-- 
Matthew Gidden
Ph.D. Candidate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192